graphite_api/render/glyph.py

```diff
@@ -12,7 +12,12 @@
 See the License for the specific language governing permissions and
 limitations under the License."""
 
-import cairocffi as cairo
+try:
+    import cairocffi as cairo
+except (NameError, ImportError, AttributeError):
```

Comment:
`AttributeError` is thrown in cases, like in certain pypy versions, where cairocffi is installed but unusable.

For example on my dev system with pypy 2.2.1:
```
python --version
Python 2.7.3 (2.2.1+dfsg-1ubuntu0.3, Sep 30 2015, 15:18:40)
[PyPy 2.2.1 with GCC 4.8.4]

python -c 'import cairocffi'
Traceback (most recent call last):
  File "app_main.py", line 72, in run_toplevel
  File "app_main.py", line 578, in run_it
  File "<string>", line 1, in <module>
  File "/home/panos/graphite-api/pypyenv/site-packages/cairocffi/__init__.py", line 20, in <module>
    from ._ffi import ffi
  File "/home/panos/graphite-api/pypyenv/site-packages/cairocffi/_ffi.py", line 4, in <module>
    ffi = _cffi_backend.FFI('cairocffi._ffi',
AttributeError: 'module' object has no attribute 'FFI'
```

`NameError` I think is not needed anymore, was using it in an older revision.